aptos-move/e2e-move-tests/src/harness.rs

```diff
@@ -803,6 +817,13 @@ impl MoveHarness {
     }
 }
 
+pub fn create_random_key_pair() -> Account {
+    let mut rng = StdRng::from_seed(OsRng.gen());
+    let privkey = Ed25519PrivateKey::generate(&mut rng);
+    let pubkey = privkey.public_key();
+    Account::with_keypair(privkey, pubkey)
```

Comment:
What's the reason to do this as opposed to e.g. Account::new_from_seed() if seed is important, or new()?